### ISSUE TITLE
Add weights for specific entities in the model??

### ISSUE THREAD

**Issue by KarlosMuradyan** [Franck-Dernoncourt/NeuroNER#125]
If there is some way, how the code should be modified to give some specific entities more weights than the others?
The thing that I want to do is giving more loss to 0 entities (Entities that are not labeled in the training set) than the others in order to predict more entities for the input. The reason for this is that lots of entities in the training set are not labeled, so  for example if "Google" is not labeled, model may not know that it should be labeled as an "Organization". Therefore I am trying to give more loss to the words without labels in order it tend to predict something. 
The inspiration came from this:
https://www.tensorflow.org/api_docs/python/tf/losses/softmax_cross_entropy
that uses weights to regulate it. I tried to change some things but it didn't work.

Thanks in advance.
